lib/runner/variant-reassignment.js

```diff
@@ -629,4 +643,12 @@ export default class VariantReassignment {
       .then(_.compact)
   }
 
+  _getDefaultErrorCallback () {
+    return (error, productDraft) => {
+      this.logger.error(`Error when processing productDraft ${JSON.stringify(productDraft)}, ` +
+        `skipping the product draft.`, error)
```

Comment:
Unfortunately this will not log out request/response and headers of the error object.

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/61/commits/35a0355701b85a2bb62e057e1f9f0275b34bb020